kde-apps/konsole/files/konsole-22.04.2-musl_malloc_trim.patch

```diff
@@ -0,0 +1,88 @@
+From b8c90a830ceaf293e61f6cd5217bb3e584f997b8 Mon Sep 17 00:00:00 2001
```

Comment:
link to commits/MRs/both at top of patches too as it saves digging in commit history